src/main/java/hu/bme/mit/spaceship/GT4500.java

```diff
@@ -20,6 +20,45 @@ public boolean fireLaser(FiringMode firingMode) {
     return false;
   }
 
```

Comment:
Write some more comments for the next task. (Because why not.... comments are fiiiine....)